modules/targetanalysis/include/target_analyzer.h

```diff
@@ -76,7 +100,65 @@ class TargetAnalyzer {
          *        
          * @return vector containing the unique Targets which have been analyzed
          */
-        vector<Target *> get_processed();
+        //vector<Target *> get_processed();
+        
+        /**
+         * @brief   Determines the longitude and latitude of the corners of the
+         *          image, assuming the geolocation provided in the telemetry is the
+         *          centroid of the photo. This function takes into account
+         *          possible rotations of the photo from the heading of the
+         *          aircraft.
+         *          
+         * @param   cameraAlpha Alpha_X and Alpha_Y values for the horizontal and vertical
+         *          values for the camera viewing (half-)angle. Measured from
+         *          the center of the photo to the outer most edge of the camera
+         *          image.
+         * @param   longitude The GPS longitude of the aircraft when the photo was
+         *          taken.
+         * @param   latitude The GPS latitude of the aircraft when the photo was
+         *          taken.
+         * @param   altitude The altitude of the aircraft from where it took the
+         *          photo.
+         * @param   heading The heading (GPS heading, not magnetic heading) from
+         *          where the aircraft took the photo.
+         * @param   img The image data for which the GPS coordinates are
+         *          calculated for. 
+         *        
+         * @return  vector containing the unique Targets which have been analyzed
+         */
+        void getGPSCorners(cv::Point2d cameraAlpha, double longitude, double latitude,
```

Comment:
Good point about using the center rather than corners. Using something similar to "pixel distance" will be used to determine other points based on the center point.